src/milia/api/async_export.cljs

```diff
@@ -76,9 +76,14 @@
    :include-hxl? :include-images? :remove-group-name? :version :query :export_id
    :include-labels? :labels-only?])
 
+(defn- get-param [key value]
+  (cond
+    (= key "_version") (str "&query='{\"" key "\":" \" value "\"}'")
+    (or value (= value false)) (str "&" key "=" value)))
```

Comment:
This should be just 
```clojure
(defn- get-param [key value]
  (cond
    (= key "_version") (str "&query='{\"" key "\":" \" value "\"}'")
    (str "&" key "=" value)))
```
because `(or value (= value false))` has already evaluated when this is called `get-param`